analysis_options.yaml

```diff
@@ -5,3 +5,4 @@ analyzer:
     body_might_complete_normally_nullable: ignore
     depend_on_referenced_packages: ignore
     no_leading_underscores_for_library_prefixes: ignore
+    use_build_context_synchronously: ignore
```

Comment:
Which place leads to the error you want to mute?

Reply:
It lints on line https://github.com/Iteo/hooked_bloc/blob/c7d5b50c356d97954bdc2c78068b7a77476832cd/lib/src/bloc_listener.dart#L34
and 
https://github.com/Iteo/hooked_bloc/blob/c7d5b50c356d97954bdc2c78068b7a77476832cd/lib/src/bloc_comparative_listener.dart#L36

I muted because I think it is the user's responsiblity to check for context state when listener called by hooked_bloc. It is better than silently ignore event when context unmounted.

Reply:
Okay, thank you for the explanation